Mergin/utils.py

```diff
@@ -186,3 +187,17 @@ def send_logs(username, logfile):
         return log_file_name, None
     except (HTTPError, URLError) as e:
         return None, str(e)
+
+
+def validate_mergin_url(url):
+    url_ping = urllib.parse.urljoin(url, urllib.parse.quote("/ping"))
+    try:
+        with urllib.request.urlopen(url_ping) as f:
+            response = json.load(f)
+            if response.get('service') != 'Mergin':
+                return 'URL is invalid Mergin URl.'
+            if response.get('status') != 'online':
+                return 'Mergin server is offline.'
+    except:
```

Comment:
Catch-all "except" is a bad practice - please specify what exception should be caught